src/services_models/reserves.jl

```diff
@@ -19,6 +20,20 @@ function activeservice_variables!(
     return
 end
 
+function activerequirement_variables!(
+    psi_container::PSIContainer,
+    devices::IS.FlattenIteratorWrapper{SR},
+) where {SR <: PSY.Reserve}
```

Comment:
This should be dispatched on the specific PSY reserve type that uses this.

Reply:
That make sense, will make that change